src/pages/Character/Character.scss

```diff
@@ -0,0 +1,38 @@
+.characterProfile {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  img {
+    margin-right: 30px;
+  }
+}
+
+.characterInfo {
+  width: 100%;
+  h1 {
+    display: block;
+    margin-bottom: 25px;
+  }
+  hr {
+    width: 100%;
+  }
+  p {
+    font-size: 1.2em;
+  }
+}
```

Comment:
Nice nesting to group SASS code for  elements inside the same container element.